INZFS.MVC/Models/DynamicForm/MultiSelectInputModel.cs

```diff
@@ -17,9 +17,9 @@ public class MultiSelectInputModel : BaseModel, IValidatableObject
 
         protected override IEnumerable<ValidationResult> ExtendedValidation(ValidationContext validationContext)
         {
-            if (Mandatory == true  && MarkAsComplete)
+            if (Mandatory == true && MarkAsComplete)
```

Comment:
With && MarkAsComplete causes a null reference error when no option is selected. Remove or develop a new solution to handle null references.

Reply:
Null Error Fixed with try catch